presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-cassandra/health-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eo pipefail
+
+# https://github.com/docker-library/healthcheck/blob/master/cassandra/docker-healthcheck
+host="$(hostname --ip-address || echo '127.0.0.1')"
```

Comment:
The link doesn't explain why not just 127.0.0.1 always. Any reason?

Reply:
maybe cassandra isn't bound to 127.0.0.1 ? I can investigate that